contour.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: contour
```

Comment:
should this be a version stream?  I "think" if you create the lowest in support version stream i.e 
```
package:
  name: contour-1.27
  version: 1.27.4
```

To match [Contour v1.27.4](https://github.com/projectcontour/contour/releases/tag/v1.27.4) then automation should auto create PRs for the .28 and .29 streams.

Reply:
the lowest version stream would be 1.27.x and the latest patch version of that stream is 4 so the version of that package should be 1.27.4, I agree with @rawlingsj here.

Reply:
we should also create a [package-version-metadata](https://github.com/chainguard-dev/package-version-metadata/) for contour

Reply:
Done, also created the same for that @developer-guy, will refactor it once we have contour in eol https://github.com/chainguard-dev/package-version-metadata/pull/551